GCTC-NTGC/gc-digital-talent
# Feature - 3164 - Public Pool Advertisement Page

Resolves #3164 

## Summary

This creates the public facing page for pool advertisements.

## Testing

1. Navigate to `/browse/pools`
2. Click on one of the pools
3. Add `/view` to the end of the path
4. Confirm you can see the page and it matches the prototype

### Testing Notes

#### Missing Cypress Tests

We are still waiting on mutations to be able to test this with cypress.

#### Status

We have the following statuses and you should expect the following when you encounter them:

- DRAFT
  - 404 if logged in as applicant
  - View normally as admin
- PUBLISHED
  - See as admin or applicant
  - Can apply as applicant
  - Tile with info in header should indicate number of days until it expires
    - Tomorrow and Today are replaced by number of days when appropriate
- EXPIRED
  - See as admin or applicant
  - Tile indicates the post is expired
  - Apply buttons should be disabled

> We will have ARCHIVED and CLOSED eventually but those have not been added yet so are not reflected.
